config.cc

```diff
@@ -62,6 +76,19 @@ namespace
     return result;
   }
 
+  QDir getDataDir() throw()
+  {
+  #if QT_VERSION >= QT_VERSION_CHECK( 5, 0, 0 )
+      QDir dir = QDir( QStandardPaths::writableLocation( QStandardPaths::GenericDataLocation ) );
+  #else
+      QDir dir = QDir( QDesktopServices::storageLocation( QDesktopServices::GenericDataLocation ) );
```

Comment:
https://doc.qt.io/archives/qt-4.8/qdesktopservices.html#StandardLocation-enum: no `QDesktopServices::GenericDataLocation`. Limit the code that calls this function to Qt 5 and return `getHomeDir()` if Qt version is less than 5 here.